pkg/collector/py/check.go

```diff
@@ -80,6 +81,7 @@ func (c *PythonCheck) Run() error {
 	}
 
 	s.Commit()
+	batcher.GetBatcher().SubmitComplete(c.ID())
```

Comment:
if the check run produced errors or it is not able to retrieve the `sender` instance it fails; shouldn't we complete anyway if one of these things happens ?

Reply:
agreed, i moved this to just after the run